src/applications/personalization/dashboard-2/tests/e2e/messaging-error.cypress.spec.js

```diff
@@ -11,14 +11,14 @@ describe('MyVA Dashboard - Messaging', () => {
     beforeEach(() => {
       mockLocalStorage();
       const mockUser = makeUserObject({
-        messaging: true,
+        messaging: false,
```

Comment:
Dumb question: What is `messaging` in this context, and why does it need to be changed from `true` to `false`?

Reply:
This test started failing due to the Cypress upgrade, and it may be a bug in the UI itself and not just an issue with the test. An `undefined` appears in the UI when running this test with the latest version of Cypress. `messaging` is used in the UI code to render different components. I'm not familiar with this feature to determine why this fixed the failure, so I'm going to get a review from the team that owns this test before I merge.

Reply:
I'll look into this.

Reply:
@erikphansen is it fine if I disable this test until you revisit it? This PR may have exposed a UI bug here. I'm seeing an `undefined` in the UI:
<img width="324" alt="Screen Shot 2021-05-21 at 8 40 43 AM" src="https://user-images.githubusercontent.com/3535749/120382755-a79a2e00-c2e9-11eb-922f-a0e3fe660b25.png">

Reply:
yes, feel free to disable it